src/doc/Document.ts

```diff
@@ -192,12 +193,12 @@ export class Document<T = unknown> {
    * Convert any value into a `Node` using the current schema, recursively
    * turning objects into collections.
    */
-  createNode(value: unknown, options?: CreateNodeOptions): Node
-  createNode(
-    value: unknown,
+  createNode<TValue = unknown>(value: TValue, options?: CreateNodeOptions): NodeType<TValue>
+  createNode<T = unknown>(
+    value: T,
```

Comment:
Would be good to keep the same generic type identifier across function signatures.

Reply:
Sorry, it was a part of attempted generic parameter rename that I committed by mistake. Reversed.